### PR TITLE
Add command to show result, keeping focus in results buffer

### PR DESCRIPTION
**Pull request by rudi**
I find this useful to jump through results by pressing `N` `O` `N` `O` ...

### CODE REVIEW

**Comment by Wilfred** [comment]
Have you tried `M-x next-error-follow-mode`? You can then just use `n` repeatedly, saving a precious keystroke :)

I'm happy to merge this if you still think it's useful, let me know what you think.

**Comment by iquiw** [comment]
Looks #66 is same request as this.

**Comment by rudi** [comment]
Cool - didn't know about `next-error-follow-minor-mode`, this is neat.  I've added it to `deadgrep-mode-hook`.  In light of #66, this could still be merged - your choice, I"m happy either way.

File: deadgrep.el [hunk 1]
```diff
@@ -1075,6 +1076,13 @@ If POS is nil, use the beginning position of the current line."
   (interactive)
   (deadgrep--visit-result #'find-file-other-window))
 
+(defun deadgrep-show-result-other-window ()
```
Comment (s-kostyaev) [inline deadgrep.el]: `next-error-no-select`?

**Comment by NightMachinary** [comment]
That `next-error-follow-minor-mode` should be added to the readme, it's fantastic!

**Comment by NightMachinary** [comment]
Is there a way to make `next-error-follow-minor-mode` center the other buffer as it moves?

**Comment by Wilfred** [comment]
Closing in favour of next-error-follow-mode. Thanks for the PR :)

**Pull request closed without merging**